scheduler/config.edn

```diff
@@ -40,7 +40,11 @@
        :keystore-path #config/env "COOK_KEYSTORE_PATH"
        :keystore-type "pkcs12"
        :keystore-pass "cookstore"}
- :rate-limit {:user-limit-per-m 1000000}
+ :rate-limit-expire-min 120 ; Expire unused rate limit entries after 2 hours.
+ :rate-limit {:user-limit-per-m 1000000
+              :job-creation {:is-enforcing false
+                             :bucket-size 15000
+                             :rate-per-minute 5003}}
```

Comment:
Because I thought rate-per-minute is convenient for us to think about. Tokens are earned continuously; buckets are refilled lazily only as needed.

`5000` the throughput limit I think we want in production, `5003` instead of `5000` is grep-bait that can be searched easily.

Reply:
Wouldn't there be other text in the logs that could help with the search rather than looking for `5003`?

Reply:
I'll set it to 5000.